components/color.css

```diff
@@ -0,0 +1,86 @@
+:root {
+  /* Primary palette */
+  --clr-blue-primary: #3753db;
+  --clr-blue-tint1: #4b64df;
+  --clr-blue-dark: #160a40;
+  --clr-white: #fff;
+  --clr-off-white: #f2f2f2;
+
+  /* secondary palette */
+  --clr-red: #e62e4d;
+  --clr-red-tint1: #e9435f;
+  --clr-green: #3ec764;
+  --clr-green-tint1: #51cd74;
+  --clr-yellow: #ffab00;
+  --clr-yellow-tint1: #ffb319;
+  --clr-teal: #00b8d9;
+  --clr-purple: #6554c0;
+  --clr-light-violet: #3d1673;
+  --clr-dark-violet: #311459;
+  --clr-highlight:#c5eed1 ;
```

Comment:
Partha make them in  capital for your code. Easy for you to read later on in variables

Reply:
Else look good to me buddy Keep on doing the good work :)
UI looks good to me simplistic and good

Reply:
Not necessary, it's about preferences.😅

Reply:
> Partha make them in capital for your code. Easy for you to read later on in variables

Yeah, but I find it easier to write the variables this way, and plus for understanding, I make sure I write "clr" in front of every color variable. I also make sure to keep clear variable names so that an outsider and get them easily. Thank you for reviewing!

Reply:
> Else look good to me buddy Keep on doing the good work :) UI looks good to me simplistic and good

Thank you!